app/resetPassword/page.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+import dynamic from "next/dynamic";
+// Components
+import {
+    TextInput,
+    Group,
+    Box,
+    Button,
+    Typography,
+    showNotification
+} from "../../components/components";
+
+// Hooks
+import { useState } from "react";
+import { useRouter } from "next/navigation";
+import { useDispatch } from "react-redux";
+
+
+import { handleSendPasswordResetEmail } from "../../firebase/auth";
+
+// export const runtime = "edge";
+
+export default function ResetPassword() {
+    const [email, setEmail] = useState("");
+    const [error, setError] = useState<boolean>(false);
+    const router = useRouter();
+    const dispatch = useDispatch();
+
+
+    const handlePasswordReset = async () => {
+        if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(email)) {
+            const response = await handleSendPasswordResetEmail(email)
+            console.log(response);
+            if (!response.error) {
+                showNotification({
+                    title: "Password Reset Email Sent.",
+                    message: "Check your inbox.",
+                });
+                router.push("/login/recruiter");
+            } else {
+                let message = "Error";
+                if (response.error == "Firebase: Error (auth/invalid-email).") {
+                    message = "Invalid Email";
+                } else if (response.error == "Firebase: Error (auth/user-not-found).") {
+                    message = "User not found"
+                }
+                showNotification({
+                    title: "Error",
+                    message: message,
+                })
+            }
+        }
+    };
+
+    return (
+        <Box
+            sx={{ maxWidth: 300 }}
+            mx="auto"
+            className="flex flex-col justify-center mt-10"
+        >
+            <Typography order={1} className="mb-1">Password Reset</Typography>
+            <Typography order={6}>Welcome to Training and Placement Cell</Typography>
```

Comment:
Do we need this welcome after the Password Reset text?

Reply:
Removed the piece of text